bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -50,6 +50,8 @@ pub enum RoomListError {
     InitializingTimeline { error: String },
     #[error("Event cache ran into an error: {error}")]
     EventCache { error: String },
+    #[error("The requested room doesn't match the membership requirements")]
+    IncorrectRoomMembership,
```

Comment:
bonus points if you include two fields `expected` / `observed` that help matching the requirements, but not mandatory for merging :)

Reply:
Noice, bonus points for me it is then! 😁